pkg/services/searchV2/bluge.go

```diff
@@ -700,4 +699,5 @@ type customMeta struct {
 	Count     uint64                  `json:"count"`
 	MaxScore  float64                 `json:"max_score,omitempty"`
 	Locations map[string]locationItem `json:"locationInfo,omitempty"`
+	SortBy    string                  `json:"sortBy,omitempty"`
```

Comment:
shouldn't we return the same string we receive so that the response makes sense on its own, without the request context? ie. sort string with `asc`/`desc` or `-` and no `-`.

Reply:
I am currently using this as in indication of which field is used for sort -- not the direction.  When a field exists with the same name in the response, we show it in the table.  I suppose a bit hacky, but 🤷